### PR TITLE
fosdem2021.md - draft

### PR DESCRIPTION
**Pull request by KBanecka**

### CODE REVIEW

File: blog/content/post/2021-01-27-fosdem2021.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+* To participate, no registration is needed. Check the [schedule](https://fosdem.org/2021/schedule/track/open_source_firmware_bmc_and_bootloader/) and choose the talks you would take part in. We are in [D.firmware](https://fosdem.org/2021/schedule/room/dfirmware/) room.
+* Find the live streaming for a D.firmware [here](https://fosdem.org/2021/schedule/streaming/). The page will be updated every few minutes to show you what is currently
+scheduled in each room.
+
+### How we are proceeding this year
+
+This online event will take place on 6 & 7 February 2021. Here you will find [the full schedule](https://fosdem.org/2021/schedule/track/open_source_firmware_bmc_and_bootloader/).
+The below points will be updated with recordings after the event.
+
+##### February 6 2020
```
Comment (macpijan) [inline blog/content/post/2021-01-27-fosdem2021.md:70]: @KBanecka 2021

File: blog/content/post/2021-01-27-fosdem2021.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+* [GRUB - Project Status Update](https://fosdem.org/2021/schedule/event/firmware_gpsu/) by [Daniel Kiper](https://fosdem.org/2021/schedule/speaker/daniel_kiper/), and [Norbert Kamiński](https://fosdem.org/2021/schedule/speaker/norbert_kaminski/)
+
+* [Open source PMCI stack implementation for add-in-card manageability.](https://fosdem.org/2021/schedule/event/firmware_ospsifam/) by [sumanthbhatt](https://fosdem.org/2021/schedule/speaker/sumanthbhatt/), and [Cezary Sobczak](https://fosdem.org/2021/schedule/speaker/cezary_sobczak/)
+
+* [Using OpenSPDM in Firmware as a Foundation for Secure Device Communication](https://fosdem.org/2021/schedule/event/firmware_uoifaaffsdc/) by [Jiewen Yao](https://fosdem.org/2021/schedule/speaker/jiewen_yao/), [xiaoyuruan](https://fosdem.org/2021/schedule/speaker/xiaoyuruan/), and [Norbert Kamiński](https://fosdem.org/2021/schedule/speaker/norbert_kaminski/)
+
+### 3mdeb on FOSDEM 2020
```
Comment (macpijan) [inline blog/content/post/2021-01-27-fosdem2021.md:98]: @KBanecka 2021

File: blog/content/post/2021-01-27-fosdem2021.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+This is another year in which, together with Daniel Kiper from Oracle, we have
+the pleasure to organize Open Source Firmware, BMC and Bootloader devroom during
+FOSDEM 2021 as a continuation of this devroom of the previous edition. We take
+part in this event to promote the use of free and open source
+software, and show how it can increase security and the quality of the code
+itself and give more business freedom outside of the vendors lock-in, from the
+point of the firmware level which is our daily bread in 3mdeb. In this blogpost
+we will gather all the information about the event and links to the videos from
+the fosdem 2020.
```
Comment (macpijan) [inline blog/content/post/2021-01-27-fosdem2021.md:32]: 2021

File: blog/content/post/2021-01-27-fosdem2021.md (hunk 1 above)
Comment (KBanecka) [reply blog/content/post/2021-01-27-fosdem2021.md:70]: done

File: blog/content/post/2021-01-27-fosdem2021.md (hunk 2 above)
Comment (KBanecka) [reply blog/content/post/2021-01-27-fosdem2021.md:98]: done

File: blog/content/post/2021-01-27-fosdem2021.md (hunk 3 above)
Comment (KBanecka) [reply blog/content/post/2021-01-27-fosdem2021.md:32]: done

File: blog/content/post/2021-01-27-fosdem2021.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+> FOSDEM is a free event for software developers to meet, share ideas and
```
Comment (miczyg1) [inline blog/content/post/2021-01-27-fosdem2021.md:34]:
```suggestion
> Free and Open source Software Developers' European Meeting (FOSDEM) is a free event for software developers to meet, share ideas and
```

File: blog/content/post/2021-01-27-fosdem2021.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+collaborate. Every year, thousands of developers of free and open source
+software from all over the world gather at the event in Brussels. In 2021, they
+will gather online.
+
+### Why Open source
+
+Before diving into the technical information around fosdem, let's make a step
```
Comment (miczyg1) [inline blog/content/post/2021-01-27-fosdem2021.md:41]:
```suggestion
Before diving into the technical information around FOSDEM, let's make a step
```
Let's keep consistent naming of the event across the blog post

File: blog/content/post/2021-01-27-fosdem2021.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+scheduled in each room.
+
+### How we are proceeding this year
+
+This online event will take place on 6 & 7 February 2021. Here you will find [the full schedule](https://fosdem.org/2021/schedule/track/open_source_firmware_bmc_and_bootloader/).
+The below points will be updated with recordings after the event.
+
+##### February 6 2020
+
+* [Open Source Firmware status on AMD platforms 2021](https://fosdem.org/2021/schedule/event/firmware_osfsoap2/) by [Piotr Król](https://twitter.com/pietrushnic) and [Marek Kasiewicz](https://fosdem.org/2021/schedule/speaker/marek_kasiewicz/)
```
Comment (miczyg1) [inline blog/content/post/2021-01-27-fosdem2021.md:72]: This schedule contains moderators: Marek, Jakub, Norbert and Cezary. They shouldn't be mentioned along with speakers. At least mark them as moderators or remove them. This is caused by the way Pentabarf interpret other roles, which is weird and incorrrect

File: blog/content/post/2021-02-02-fosdem2021.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
+---
```
Comment (miczyg1) [inline blog/content/post/2021-02-02-fosdem2021.md:1]: There are two the same files, but with different publication dates. Why?

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: blog/content/post/2021-01-27-fosdem2021.md (hunk 6 above)
Comment (KBanecka) [reply blog/content/post/2021-01-27-fosdem2021.md:72]: ok, thank you for mentioning, I will remove moderators

File: blog/content/post/2021-02-02-fosdem2021.md (hunk 7 above)
Comment (KBanecka) [reply blog/content/post/2021-02-02-fosdem2021.md:1]: I tried to change the date in the file.

Comment (KBanecka) [reply blog/content/post/2021-02-02-fosdem2021.md]: I removed duplicated file

File: blog/content/post/2021-01-27-fosdem2021.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+
+---
+### Introduction
+
+This is another year in which, together with [Daniel Kiper](https://fosdem.org/2021/schedule/speaker/daniel_kiper/) from Oracle, we have the pleasure to organize Open Source Firmware, BMC and
+Bootloader devroom during FOSDEM 2021 as a continuation of this devroom of the
+previous edition. We take part in this event to promote the use of free and open
+source software, and show how it can increase security and the quality of the
+code itself and give more business freedom outside of the vendors lock-in, from
+the point of the firmware level which is our daily bread in 3mdeb. In this
```
Comment (pietrushnic) [inline blog/content/post/2021-01-27-fosdem2021.md:27]:
```suggestion
the point of the firmware level which is our bread and butter in 3mdeb. In this
```

File: blog/content/post/2021-01-27-fosdem2021.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+project. The company that decided to develop a particular project is more eager
+to hire contributors with proven expertise.
+
+* The company can get improved performance for a relatively short amount of time.
+ Having open-source repository allows to prove the competencies and gain trust
+ without any additional audit or checks.
+
+* The source code has no value itself. Most of the products are made by modifying
+existing code components, so technology knowledge that allows performing on the
+ready-to-use code in the proposed business model brings the essential value.
```
Comment (pietrushnic) [inline blog/content/post/2021-01-27-fosdem2021.md:57]:
```suggestion
ready-to-use code in the proposed business model brings the essential value. There
are some exceptions to that like very innovative and research base solutions or
fully-featured systems, which secrecy may build value that owners do not want to give
to the community using OSS license.
```

**Review by artur-rs — APPROVED** [review]

**Pull request merged**